ETLParser.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+# Parser for PEDSnet ETL conventions document
+
+import json
+import os.path
+import re
+import sys
+
+_h1 = re.compile(r'^\#[^\#]+')
+_h2 = re.compile(r'^\#\#[^\#]+')
+
+
+class Document():
+    def __init__(self, path):
+        self.dict = {}
+        self.dict['content'] = ''
+        self.dict['tables'] = {}
+
+        with open(path, "r") as f:
+            self.lines = f.readlines()
+
+        self.get_content()
+
+        while self.pos < len(self.lines):
+            self.get_next_table()
+
+    def add_content(self, content):
+        self.dict['content'] = content
+
+    def add_table(self, table, content=''):
+        self.dict['tables'][table] = {}
+        self.dict['tables'][table]['content'] = content
+        self.dict['tables'][table]['fields'] = {}
+
+    def add_table_content(self, table, content):
+        self.dict['tables'][table]['content'] = content
+
+    def add_field(self, table, field,
+                  required, data_type, desc, conv):
```

Comment:
Only wrap arguments if the line exceeds 79 characters. I don't believe this one does.